config/locales/ja.yml

```diff
@@ -11,4 +11,4 @@ ja:
   label_fieldset_default_state: フィールドセットのデフォルト状態
   label_fieldset_state_all_expended: 全て展開
   label_fieldset_state_all_collapsed: 全て折りたたみ
-  error_can_not_delete_custom_fields_group: カスタムフィールドグループを削除できません。
+  error_can_not_delete_custom_fields_group: カスタムフィールドグループを削除できません
```

Comment:
Well, current translation comes from the following Redmine core side, so I will keep last `。` as well on Weblate side.
https://github.com/redmine/redmine/blob/5.1-stable/config/locales/ja.yml#L213
```yml
  error_can_not_delete_custom_field: 'カスタムフィールドを削除できません。'
```

Reply:
Fixed above and dismissed the warning on Weblate.